### ISSUE TITLE
Break over sound

### ISSUE THREAD

**Issue by stickynoteme** [gazugafan/RestStop#2]
This break timer is almost perfect for me, however I had to move back to workrave as there is no way to set a sound to let me know when my break is over. Which resulted in me either taking far to long of breaks or impatiently watching the clock rather then enjoying the break.

**Comment by gazugafan** [comment]
I like the idea! I haven't touched this since the initial release 2 years ago, but yeah if I find some free time I might add this feature in.

**Comment by Diazbesjorge** [comment]
I agree this is the only feature missing to fully replace other apps such as Stretchly or Workrave.
